### ISSUE TITLE
Add unit-ads library to jCenter for easier AndroidStudio integration.

### ISSUE THREAD

**Issue by townsfolk** [Applifier/unity-ads#57]
The current Android integration instructions require us to download the source or master zip files, and include them into our projects manually.

It would be easier if the ads library was uploaded to jCenter (or some other Maven repo) so we can just add it to our build.gradle files.

We would still need add the activity to our AndroidManifest files, but this would make upgrading a lot easier.

**Comment by gomsu** [comment]
True. We are looking into either setting up our own maven repo or putting ads to one that already exists to make AS integration easier.
